content/app/guides/payment/frontend-configuration/_index.nb.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Frontend konfigurasjon 
+description: Følg disse stegene for å konfigurere frontenden din for betaling.
+weight: 3
+---
+### 1. Legg til Payment layoutSet
+
+Legg til en ny layoutSet-mappe for betalingsoppgaven din, og oppdater `layout-sets.json` filen.
+
+Din `layout-sets.json` kan se slik ut:
+
+```json
+{
+  "$schema": "https://altinncdn.no/toolkits/altinn-app-frontend/4/schemas/json/layout/layout-sets.schema.v1.json",
+  "sets": [
+    {
+      "id": "form",
+      "dataType": "model",
+      "tasks": [
+        "Task_1"
+      ]
+    },
+    {
+      "id": "payment",
+      "dataType": "model",
+      "tasks": [
+        "Task_2"
+      ]
+    }
+  ]
+}
+```
+
+I din payment layoutSet mappe, legg til en ny fil, `payment.json`, med følgende layout:
+
+```json
+{
+  "$schema": "https://altinncdn.no/toolkits/altinn-app-frontend/4/schemas/json/layout/layout.schema.v1.json",
+  "data": {
+    "layout": [
+      {
+        "id": "payment-test-component",
+        "type": "Payment",
+        "textResourceBindings": {
+          "title": "Oppsummering"
+        }
+      }
+    ]
+  }
+}
+```
+
+Dette er nødvendig for at betaling skal fungere. Uten dette vil betalingssteget ditt bare vise en hvit side.
```

Comment:
For at betaling skal fungere, må du ha med dette. Ellers vil betalingssteget ditt bare vise en hvit side.